torch/_inductor/config.py

```diff
@@ -692,6 +692,11 @@ class cpp:
         == "1"
     )
 
+    # Disable the tiling select heuristic
+    enable_tiling_heuristics = (
+        os.environ.get("TORCHINDUCTOR_CPP_ENABLE_TILING_SELECT_HEURISTIC", "1") == "1"
```

Comment:
nit: make naming consistent:
```suggestion
        os.environ.get("TORCHINDUCTOR_CPP_ENABLE_TILING_HEURISTIC", "1") == "1"
```